index.js

```diff
@@ -22,8 +22,9 @@ module.exports = {
 
   treeForAddon: function(tree) {
     var WorkerManifest = require('./lib/worker-manifest');
+    var rootURL = this.project.config(this.env).rootURL;
```

Comment:
Is `this.env` available on `Addon` instances? I don't see it [in the documentation](https://ember-cli.com/api/classes/Addon.html). An alternative might be `this.project.config(EmberApp.env())`, where `EmberApp = require('ember-cli/lib/broccoli/ember-app')`

Reply:
@dfreeman sorry about that. Updated!

Reply:
@dfreeman Oops. Updated, but having trouble with the build.